src/ducks/reimbursements/Caption.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import cx from 'classnames'
+
+import styles from 'ducks/reimbursements/Reimbursements.styl'
+
+const Caption = props => {
+  const { className, ...rest } = props
+
+  return <p className={cx(styles.Caption, className)} {...rest} />
```

Comment:
let's add a test on this new component please to document its usage.

Reply:
vu ce qu'il fait je ne pense pas que ce soit la peine non